src/types.jl

```diff
@@ -160,6 +160,21 @@ struct RGB4{T<:Fractional} <: AbstractRGB{T}
 end
 RGB4(r::T, g::T, b::T) where {T<:Fractional} = RGB4{T}(r, g, b)
 
+"""`RGB565` is Red-Green-Blue colorspace represented by two bytes (16 bits).
+
+1        |1          |        0
+5 4 3 2 1|0 9 8 7 6 5|4 3 2 1 0 
+---------|-----------|---------
+R R R R R|G G G G G G|B B B B B
+"""
+struct RGB565 <: AbstractRGB
```

Comment:
```suggestion
struct RGB565 <: AbstractRGB{Normed}
```

Or make it `AbstractRGB{N2f6}` and use `N2f6` throughout (in both constructors and `red`/`green`/`blue`).